cmd/hlb/command/run.go

```diff
@@ -175,18 +175,17 @@ func Run(ctx context.Context, cln *client.Client, uri string, info RunInfo) (err
 	}
 	ctx = local.WithOs(ctx, info.Os)
 	ctx = local.WithArch(ctx, info.Arch)
+	var progressOpts []solver.ProgressOption
 	if info.DefaultPlatform != "" {
 		platformParts := strings.SplitN(info.DefaultPlatform, "/", 2)
 		if len(platformParts) < 2 {
 			return fmt.Errorf("Invalid platform specified: %s", info.DefaultPlatform)
 		}
 		ctx = codegen.WithDefaultPlatform(ctx, specs.Platform{OS: platformParts[0], Architecture: platformParts[1]})
+		progressOpts = append(progressOpts, solver.WithLogPrefix(info.DefaultPlatform))
```

Comment:
Perhaps we should allow for `LogPrefix []string` in the RunInfo so we can pass them in from our tools in case we want to do something special with the logging.  Then maybe if the `(*RunInfo).LogPrefix` is empty we automatically add the default platform to the prefix?

Reply:
I guess an alternative to this would be to allow a `solver.Progress` to be passed into the `RunInfo` instead of just prefixes, not sure which option makes more sense.

Reply:
I like making `LogPrefixes` explicit, updated the PR to support it.